src/EmployeeBook.java

```diff
@@ -0,0 +1,242 @@
+public class EmployeeBook {
+    private final Employee[] employees = new Employee[10];
+
+    public void fillEmployees() {
+        employees[0] = new Employee("Иванов", "Иван", "Иванович", 1, 55000);
+        employees[1] = new Employee("Иванова", "Дарья", "Петровна", 1, 55000);
+        employees[3] = new Employee("Кузнецова", "Татьяна", "Михайловна", 3, 85000);
+        employees[4] = new Employee("Сорокин", "Виктор", "Дмитриевич", 3, 67000);
+        employees[5] = new Employee("Петров", "Петр", "Петрович", 4, 35000);
+        employees[6] = new Employee("Долгая", "Ольга", "Васильевна", 5, 78000);
+        employees[7] = new Employee("Ванютин", "Иван", "Петрович", 5, 95000);
+        employees[8] = new Employee("Тихонова", "Наталья", "Ивановна", 5, 115000);
+        employees[9] = new Employee("Черный", "Александр", "Владимирович", 5, 115000);
+    }
+
+    public Employee[] getEmployees() {
+        return employees;
+    }
+
+    public void printAllEmployees(Employee[] employees) {
```

Comment:
Массив employess - это свойство класса EmployeeBook. Он доступен всем методам класса, отдельно его передавать как параметр не нужно

```suggestion
    public void printAllEmployees() {
```
Это актуально для всех методов